client/src/view/AssignedDriver.js

```diff
@@ -0,0 +1,93 @@
+import React from "react"
+import styled from "styled-components";
+import Destination from "./icons/Destination";
+import Phone from "./icons/Phone";
+import Calendar from "./icons/Calendar";
+import GoogleRoutesForm from "./forms/GoogleRoutesForm";
+import Vertical from "./icons/Vertical";
+import Avatar from './Avatar';
+
+const WrapperTop = styled.div`
+	font-size: 20px;
+	font-weight: bold;
+	color: gray;
+	padding-top: 20px;
+	width: 70%;
+	position: absolute;
+`
+const Wrapper = styled.div`
+
+`
+const DivLeft = styled.div`
+	float: left;
+`
+
+const Map = styled.div`
+	float: right;
+	border: 1px solid black;
+`
+
+const AvailableDiv = styled.div`
+	width: 70%;
+	height: 17%;
+	position: absolute; 
+	top: 217px;
+	font-weight: bold;
+	background-color: #f2f2f2; 
+`
+
+const ReplyDiv = styled.div`
+	width: 70%;
+	height: 65px;
+	position: absolute; 
+	top: 307px;
+	padding-top: 10px;
+	background-color: #e6e6e6;
+`
+
+const Reply = styled.p`
+	font-size: 12px;
+`
+const Status = styled.p`
+	font-size: 12px;
+	display: block;
+	padding-left: 60px;
+	position: absolute;
+	top: 30px;
+	
+`
+
+
+export default () => (
+<Wrapper>
+	<WrapperTop>
+		<DivLeft>
+			<Vertical />Cobbs Hill Park<br/><br/>
```

Comment:
You can reference variables in JSX by putting them inside curly braces like [here](https://github.com/mxs4321/WASPS-Scheduling-System/blob/master/client/src/view/FindUsers.js#L40)